guides/release/pages.yml

```
@@ -1,325 +1,325 @@
 - title: "Introduction"
   url: "toc-heading_introduction"
   is_heading: true
-- title: "Guides and Tutorials"
+- title: "Guides et Tutoriels"
   url: "index"
   skip_toc: true
   pages:
-    - title: "Ember.js Guides"
+    - title: "Guides Ember.js"
       url: ""
-- title: "Getting Started"
+- title: "Commencer"
   url: "getting-started"
   pages:
-    - title: "How To Use The Guides"
+    - title: "Utiliser le Guide"
       url: "index"
-    - title: "Quick Start"
+    - title: "Démarrage rapide"
       url: "quick-start"
-    - title: "Working with HTML, CSS, and JavaScript"
+    - title: "Travailler avec HTML, CSS, et JavaScript"
       url: "working-with-html-css-and-javascript"
-    - title: "Anatomy of an Ember App"
+    - title: "Anatomie d'une App Ember"
       url: "anatomy-of-an-ember-app"
-- title: "Tutorial"
+- title: "Tutoriel"
   url: "tutorial"
   pages:
-    - title: "Part 1"
+    - title: "Partie 1"
       url: "part-1"
       pages:
         - title: "Introduction"
           url: "index"
         - title: "Orientation"
           url: "orientation"
-        - title: "Building Pages"
+        - title: "Construire des Pages"
           url: "building-pages"
-        - title: "Automated Testing"
+        - title: "Tests Automatisés"
           url: "automated-testing"
-        - title: "Component Basics"
+        - title: "Composants Basiques"
           url: "component-basics"
-        - title: "More About Components"
+        - title: "Plus sur les Composants"
           url: "more-about-components"
-        - title: "Interactive Components"
+        - title: "Composants Interactifs"
           url: "interactive-components"
-        - title: "Reusable Components"
+        - title: "Composants Réutilisables"
           url: "reusable-components"
-        - title: "Working With Data"
+        - title: "Travailler avec des Données"
           url: "working-with-data"
-        - title: "Recap"
+        - title: "Récap"
           url: "recap"
-    - title: "Part 2"
+    - title: "Partie 2"
       url: "part-2"
       pages:
         - title: "Introduction"
           url: "index"
-        - title: "Route Params"
+        - title: "Paramètres des Routes"
           url: "route-params"
-        - title: "Service Injection"
+        - title: "Injection de Service"
           url: "service-injection"
         - title: "Ember Data"
           url: "ember-data"
-        - title: "Provider Components"
+        - title: "Composants Provider"
```

What about highlighting the English word, do we want that?

```suggestion
        - title: 'Composants dit "Provider"'
```